### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by tcyrus**
Cleaned-up code + Added GitHub Actions

### CODE REVIEW

File: src/object.rs [hunk 1]
```diff
@@ -328,60 +328,59 @@ pub struct ObjectHandle {
...
+         ((4, 0x80), ObjectCapability::DeleteOpaque),
+
+         ((5, 0x01), ObjectCapability::DeleteAuthenticationKey),
+         ((5, 0x02), ObjectCapability::DeleteAsymmetricKey),
+         ((5, 0x04), ObjectCapability::DeleteWrapKey),
+         ((5, 0x08), ObjectCapability::DeleteHmacKey),
+         ((5, 0x10), ObjectCapability::DeleteTemplate),
+         ((5, 0x20), ObjectCapability::DeleteOtpAeadKey)]
+         .iter().cloned().collect();
     };
```
Comment (a-dma) [inline src/object.rs]: This doesn't build, there is an unmatched brace.

Can you please update and make sure it builds?

**Comment by tcyrus** [comment]
@a-dma Is there anything else?

**Comment by a-dma** [comment]
Sorry to be annoying, but this still doesn't build for me (this repo really need some CI).

```bash
error: expected one of `::`, `=>`, `if`, or `|`, found `as`
   --> src/object.rs:745:38
    |
745 |             lyh::YH_ORIGIN_GENERATED as u8 =>
    |                                      ^^ expected one of `::`, `=>`, `if`, or `|` here
```

**Comment by a-dma** [comment]
Hi, I don't mean to be rude, but  have you tried to build any of this? Does it build for you?

I can keep pasting compiler errors here, but I don't think it's a good way to spend anybody's time.

If this is actually building for you then please provide some more information about what version of Rust you're using and how you're building it.

**Comment by tcyrus** [comment]
Sorry about that. I fixed the build and setup CI builds via GitHub Actions to test it.

**Comment by a-dma** [comment]
Thanks. It builds now, however this has now become three things in one: syntax updates, adding the CI script and changing dependencies.

Moreover, the tests require a device so just `cargo test` won't work.

If you back out the dependency change and the yml file I'll merge the code changes.

**Comment by tcyrus** [comment]
I deleted the CI script, but the update to `lazy_static` fixes a build error.

**Comment by a-dma** [comment]
Thanks, merging.

**Pull request merged**